controllers/capabilities/policyregistry_status_reconciler.go

```diff
@@ -15,17 +15,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-type PolicyStatusReconciler struct {
+type PolicyRegistryStatusReconciler struct {
 	*reconcilers.BaseReconciler
-	resource            *capabilitiesv1beta1.Policy
+	resource            *capabilitiesv1beta1.PolicyRegistry
 	providerAccountHost string
 	policy              *threescaleapi.APIcastPolicy
```

Comment:
Should this attribute be renamed to registryPolicy?? I see the type is `threescaleapi.APIcastPolicy` but I'm not sure whether that means a Policy or a RegistryPolicy

Reply:
Yeah, `threescaleapi.APIcastPolicy` is the object resturned by the 3scale API representing an policy registry item